CIP-0050/README.md

```diff
@@ -1,431 +1,239 @@
 ---
-CIP: 50
-Title: Pledge Leverage-Based Staking Rewards
-Status: Proposed
-Category: Ledger
-Authors:
+CIP: 50 
+Title: Pledge Leverage-Based Staking Rewards 
+Category: Ledger / Protocol 
+Authors: 
   - Michael Liesenfelt <michael.liesenfelt@gmail.com>
-Implementors: N/A
-Discussions:
-  - https://github.com/cardano-foundation/CIPs/pull/242
-  - https://forum.cardano.org/t/cip-shelley-s-basho-voltaire-decentralization-update/97685
-  - https://forum.cardano.org/t/minimum-pool-fees-with-a-brief-mention-of-k-changes/97002/82
-Created: 2022-04-04
-License: CC-BY-4.0
+  - Ryan Wiley
+  - Rich Manderino [ECP]
+  - Stef M [RABIT]
+  - Wayne [OTG]
+  - Homer J (AAA)
+  - Chad [BBHMM] 
+Status: Draft (Updated) 
+Created: 4 April 2022 (original), 20 May 2025 (updated) 
+Discussion: Cardano SPO Incentives Working Group
```

Comment:
```suggestion
  - https://github.com/cardano-foundation/CIPs/pull/242
  - https://forum.cardano.org/t/cip-shelley-s-basho-voltaire-decentralization-update/97685
  - https://forum.cardano.org/t/minimum-pool-fees-with-a-brief-mention-of-k-changes/97002/82
  - https://github.com/cardano-foundation/CIPs/pull/1042/files
```
These are discussion links about the CIP, not discussion parties, so the removed material definitely needs to remain & include this PR as well.  Since discussion / agreement by the Working Group was certainly significant, maybe it can be bulleted in the Path to Active and detailed somewhere like the _Rationale_?